src/icon/exclamationIcon.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import BaseIcon from '_utils/icon'
+
+// tslint:disable:max-line-length
+export const ExclamationIcon = (props: Icon) => (
```

Comment:
Maybe the Exclamation icon "within a circle" can be challenged since we have the Warning icon which is an Exclamation within a triangle

Reply:
In fact I remember I asked a designer about that a few months ago for a different feature and they agreed.

Reply:
If we challenge that, we will have to update ios/android for consistency.
I would prefer to keep it simple for now and avoid UX/mobile frictions.

Reply:
Ok